src/libechonest.mk

```diff
@@ -0,0 +1,28 @@
+# This file is part of MXE.
+# See index.html for further information.
+
+PKG             := libechonest
+$(PKG)_IGNORE   :=
+$(PKG)_VERSION  := 2.3.1
+$(PKG)_CHECKSUM := ab961ab952df30c5234b548031594d7e281e7c9f2a9d1ce91fe5421ddde85e7c
+$(PKG)_SUBDIR   := $(PKG)-$($(PKG)_VERSION)
+$(PKG)_FILE     := $(PKG)-$($(PKG)_VERSION).tar.gz
+$(PKG)_URL      := https://github.com/lfranchi/libechonest/archive/$($(PKG)_VERSION).tar.gz
+$(PKG)_DEPS     := gcc qt qjson
+
+define $(PKG)_UPDATE
+    $(WGET) -q -O- 'https://github.com/lfranchi/libechonest/archive/' | \
+    $(SED) -n 's,.*/\([0-9][^"]*\)/"\.tar.*,\1,p' | \
+    sort | uniq | \
+    head -1
+endef
+
+define $(PKG)_BUILD
+    mkdir '$(1)/build'
+    cd '$(1)/build' && cmake .. \
```

Comment:
>Do you mean that usage works in a normal shell session without setting the $PATH?

Yes. I am building EiskaltDC++ without editing $PATH in ~/.bashrc
For example: 
https://gist.github.com/pavelvat/1af9cf37d529bf41d87e
https://gist.github.com/pavelvat/009e52249a41133ad720